src/app/lecture/services/feedback.service.ts

```diff
@@ -0,0 +1,23 @@
+import {Injectable} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {environment} from '../../../environments/environment';
+
+interface FeedbackDTO {
+  rating: number;
+  traineeId: number;
+  learningObjectId: number;
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class FeedbackService {
+
+  constructor(private http: HttpClient) {
+  }
+
+  submitRating(rating: number): void {
+    const feedback = {rating, traineeId: 1, learningObjectId: 1}; // TODO: get trainee and learning object
```

Comment:
You can use the TraineeService to retrieve the trainee id (look at the question.service to see the code).